README.md

```diff
@@ -58,4 +58,33 @@ This repository is specifically laid out to make it easy to add new features and
 
 A new version branch would only ever be created if a significant change is being made which would have a major impact on consumers of the script plugin. In most cases, this is likely because the build scripts that include the script plugin would need to change themselves, but this would also apply if there would be a fundamental change to the artifacts being produced that would need consideration before updating an app's build script.
 
-Once a new version exists, only bug fixes should ever be added to older versions. These bug fixes should be merged into newer versions if they are still applicable.
\ No newline at end of file
+Once a new version exists, only bug fixes should ever be added to older versions. These bug fixes should be merged into newer versions if they are still applicable.
+
+## Publishing
+
+If your project will be publishing an AAR, you will also need to apply the [ttm_module_publish gradle file](ttm_module_publish.gradle):
+
+```gradle
+apply from: 'https://www.github.com/geotracsystems/ttm-mep-build-scripts/raw/master/ttm_module_publish.gradle'
```

Comment:
Our examples should show a version being referenced instead of master. We don't really want anyone to reference master directly, because that could have changes that break compatibility.

Reply:
Ok should this be `version/1` or `version/3` or something else ?

Reply:
This is a separate file, so it doesn't impact compatibility for those already using version 1. As such, it can go in version 1. We will delete version 2.